api/swagger.yaml

```diff
@@ -19,10 +19,10 @@ produces:
 consumes:
   - "application/json"
   - "text/plain"
-basePath: "/v1.39"
+basePath: "/v1.40"
 info:
   title: "Docker Engine API"
-  version: "1.39"
+  version: "1.40"
```

Comment:
We'll have to bump the actual API version as well, like in; https://github.com/moby/moby/pull/38003/files#diff-c52a35abd674173ebde0a817d9e4e784R6